tayo-blog/src/App.css

```diff
@@ -0,0 +1,124 @@
+@import url('https://fonts.googleapis.com/css2?family=Chelsea+Market&family=Roboto:wght@300;400;500;700;900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+}
+
+.header {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin-bottom: 50px;
+  border-bottom: 1px solid red;
+  padding-bottom: 15px;
+}
+
+.nav-title h1 {
+  font-family: 'Chelsea Market', cursive;
+  font-size: 2.8rem;
+  font-weight: 400;
+}
+.nav-title span {
+  color: red;
+  font-weight: 800;
+}
+
+.nav-links {
+  text-decoration: none;
+  margin-left: 50px;
+  color: #000;
+  font-family: 'Roboto', sans-serif;
+  font-size: 1rem;
+  font-weight: 500;
+}
+
+.blog-container,
+.post-container {
+  width: 60%;
+  margin: 0 auto;
+}
+.blog-body {
+  margin-bottom: 35px;
+  /* border-bottom: 1px solid #000; */
+}
+
+.blog-header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.blog-header h2 {
+  font-family: 'Roboto', sans-serif;
+  color: lightblue;
+  margin-bottom: 10px;
+}
+
+.blog-body p {
+  text-align: justify;
+  line-height: 1.5;
+}
+
+.blog-body .read-more {
+  color: blue;
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.post-container form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.post-container h3 {
+  font-family: 'Chelsea Market', cursive;
```

Comment:
Put this font family into a CSS variable and reuse that, instead of always writing this.